components/Background/index.jsx

```diff
@@ -0,0 +1,12 @@
+import Image from 'next/image';
+
+const Background = ({ imagePath, children }) => {
+  return (
+    <div className="h-screen w-screen">
+      <Image alt="background" src={imagePath} layout="fill" objectFit="cover" quality={100} />
+      <div className="h-full backdrop-blur-3xl bg-shadow flex flex-col justify-center items-center">{children}</div>
```

Comment:
tried to use utility classes as much as possible throughout the PR - the tailwind way!